tests/test_models.py

```diff
@@ -2235,6 +2239,7 @@ def setUp(self):
         self.user = UserFactory()
         self.auth = Auth(user=self.user)
         self.project = ProjectFactory(creator=self.user, description='foobar')
+        ensure_licenses()
```

Comment:
Is this call necessary? The `init_app` call in `OsfTestCase` should take care of this, unless I'm missing something.